apps/src/lib/levelbuilder/rubrics/EvidenceDescriptions.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Heading6} from '@cdo/apps/componentLibrary/typography';
+import EvidenceDescriptionsRow from './EvidenceDescriptionsRow';
+
+export default function EvidenceDescriptions({isAiEnabled}) {
+  return (
+    <div>
+      <div style={styles.grid}>
+        <Heading6 style={styles.columnHeaders}>Evidence level</Heading6>
+        <Heading6 style={styles.columnHeaders}>
+          Description for external viewers
+        </Heading6>
+        <Heading6 style={styles.columnHeaders}>
+          Description for AI generated evaluation
+        </Heading6>
+      </div>
+      <EvidenceDescriptionsRow
+        isAiEnabled={isAiEnabled}
+        evidenceLabel={'Extensive Evidence'}
```

Comment:
I wonder if we should have these in a constant somewhere. I don't think you necessarily need to do that in this PR but it might make saving easier when you get there